install.js

```diff
@@ -304,6 +305,14 @@ function requestBinary(requestOptions, filePath) {
   return deferred.promise
 }
 
+function getTargetPlatform(){
+  if (process.env.PHANTOMJS_PLATFORM){
```

Comment:
i did a search for npm_config_platform and couldn't find anything, can you link documentation, or where the idiom is outlined?